dymos/transcriptions/common/test/test_boundary_constraint_comp.py

```diff
@@ -8,6 +8,10 @@
 
 from dymos.transcriptions.common.boundary_constraint_comp import BoundaryConstraintComp
 
+# Modify class so we can run it standalone.
+from dymos.utils.misc import CompWrapperConfig
+BoundaryConstraintComp = CompWrapperConfig(BoundaryConstraintComp)
```

Comment:
+1 for CompWrapperConfig...nice 👍

Reply:
Just tried to think "What would Bret do?"